homeassistant/components/websocket_api/const.py

```diff
@@ -21,9 +21,12 @@
 
 DOMAIN: Final = "websocket_api"
 URL: Final = "/api/websocket"
-PENDING_MSG_PEAK: Final = 512
+PENDING_MSG_PEAK: Final = 1024
 PENDING_MSG_PEAK_TIME: Final = 5
-MAX_PENDING_MSG: Final = 2048
+# Maximum number of messages that can be pending at any given time.
+# This is effectively the upper limit of the number of entities
+# that can fire state changes within ~1 second.
+MAX_PENDING_MSG: Final = 4096
```

Comment:
I think it makes sense to raise these limits since the system can now handle 4096 entities firing a state change at start